archai/nlp/nvidia_transformer_xl/olx_small.yaml

```diff
@@ -0,0 +1,104 @@
+# Base
```

Comment:
Wondering why we need new yaml. Can we inherit from existing sections and use existing yaml?